packages/expo/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### 🛠 Breaking changes
 
+- Drop `Logs` module export. ([#18596](https://github.com/expo/expo/pull/18596) by [@EvanBacon](https://github.com/EvanBacon))
+- Drop support for `logUrl` endpoint (legacy `expo-cli` logging will no longer work). ([#18596](https://github.com/expo/expo/pull/18596) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
Could you add a bit more user-friendly explanation to this changelog item to make it clear to use npx expo instead, which supports the new logging protocol?